### ISSUE TITLE
Pressing Enter on first line inserts */ even if comment already closed

### ISSUE THREAD

**Issue by mikkotikkanen** [peterflynn/reasonable-comments#5]
Personally, I use block comments like this. So, when I press enter after the main comment, this...

```
/* Block comment |
 * ------------------------------------------------------------------------------------------ */
```

...becomes this...

```
/* Block comment
 * |
 */
 * ------------------------------------------------------------------------------------------ */
```

...which obviously breaks things. :/

**Comment by peterflynn** [comment]
Clarifying title -- the fundamental issue is that hitting Enter on the first line of an already-closed comment will _always_ insert an extra `*/`

**Comment by peterflynn** [comment]
Fixed in version 1.2!
